Dockerfile

```diff
@@ -12,93 +12,90 @@ FROM chef AS planner
 COPY . .
 RUN cargo chef prepare --recipe-path recipe.json
 
-FROM chef AS cacher
+FROM chef AS builder
```

Comment:
I was concerned about removing this cacher step as it was recommended by `cargo chef`, but it seems that was older guidance that was removed a while ago, thanks for catching this! https://github.com/LukeMathWalker/cargo-chef/pull/90